src/tracks/playabletrack/notetrack/ui/NoteTrackAffordanceControls.cpp

```diff
@@ -0,0 +1,83 @@
+#include "NoteTrackAffordanceControls.h"
```

Comment:
Remembering the rule that a .cpp file's own header should be the first included, good